PoGo.NecroBot.Logic/State/Session.cs

```diff
@@ -202,6 +202,31 @@ public bool ReInitSessionWithNextBot(Account bot = null, double lat = 0, double
             var session = TinyIoCContainer.Current.Resolve<ISession>();
 
             var nextBot = manager.GetSwitchableAccount(bot);
+
+            //var session = TinyIoCContainer.Current.Resolve<ISession>();
+            var Account = !string.IsNullOrEmpty(nextBot.Nickname) ? nextBot.Nickname : nextBot.Username;
+            var TotXP = 0;
+
+            for (int i = 0; i < nextBot.Level + 1; i++)
+            {
+                TotXP = TotXP + Statistics.GetXpDiff(i);
+            }
+
+            long? XP = nextBot.CurrentXp;
+            if (XP == null) { XP = 0; }
+            long? SD = nextBot.Stardust;
+            if (SD == null) { SD = 0; }
+            var NLevelXP = nextBot.NextLevelXp;
+            if (nextBot.NextLevelXp == null) { NLevelXP = 0; }
+
+            Logger.Write($"Account changed to {Account}", LogLevel.BotStats);
+
+            if (session.LogicSettings.NotificationConfig.EnablePushBulletNotification == true)
+                PushNotificationClient.SendNotification(session, $"Account changed to", $"{Account}\n" +
```

Comment:
I do not know if it was @TheWizard1328  that modified it but I tested it and it's good